addons/web/static/src/legacy/js/views/list/list_editable_renderer.js

```diff
@@ -784,8 +784,8 @@ ListRenderer.include({
             return widget.widthInList;
         }
         const field = this.state.fields[name];
-        if (!field) {
-            // this is not a field. Probably a button or something of unknown
+        if (!field || fieldsInfo[name].widget == "color_picker") {
```

Comment:
you probably could just set `widthInList` to 1 in the widget to avoid having a special case here in the list renderer. 
That would be cleaner